bot/seasons/evergreen/showprojects.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from discord.ext import commands
+
+from bot.constants import Channels
+
+log = logging.getLogger(__name__)
+
+
+class ShowProjects(commands.Cog):
+    """Cog that reacts to posts in the #show-your-projects"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.lastPoster = 0  # Given 0 as the default last poster ID as no user can actually have 0 assigned to them
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        """Adds reactions to posts in #show-your-projects"""
+
+        reactions = ["\U0001f44d", "\U00002764", "\U0001f440", "\U0001f389", "\U0001f680", "\U00002b50", "\U0001f6a9"]
+        if message.channel.id == Channels.show_your_projects and message.author.id != 528937022996611082 \
```

Comment:
I'd personally prefer if this checked whether the author is a bot rather than just having SeasonalBot's ID hard coded in. However if we do decide to keep it like this, we should have a comment to explain that this is SeasonalBot's ID, or better yet name it as a variable or constant so there's no confusion.  

But ideally, it should just check to see if it's a bot or not.

You might also consider:
```suggestion
        if (message.channel.id == Channels.show_your_projects
            and message.author.id != 528937022996611082
            and message.author.id != self.lastPoster):
```
Or alternatively 
```suggestion
        if (message.channel.id == Channels.show_your_projects 
            and message.author.id not in (self.lastPoster, 528937022996611082):
```

Reply:
You don't need an ID at all to check for itself. `User` and `Member` objects both support direct equality checks between themselves & the other.

Reply:
I sweat d.py has something like isbot? I can't remember. Will need to check docs when on PC.